lib/src/main/java/graphql/nadel/engine/transform/NadelServiceTypeFilterTransform.kt

```diff
@@ -117,25 +122,6 @@ class NadelServiceTypeFilterTransform : NadelTransform<State> {
         )
     }
 
-    private fun isTypeOwnedByService(
-        objectTypeName: String,
-        executionContext: NadelExecutionContext,
-        service: Service,
-        executionBlueprint: NadelOverallExecutionBlueprint,
-    ): Boolean {
-        val typeNamesOwnedByService = executionBlueprint.getOverAllTypeNamesForService(service)
-        // Add underlying type names as well to handle combination of hydration and renames.
-        // Transforms are applied to hydration fields as well, and those fields always reference
-        // elements from the underlying schema
-        val underlyingTypeNamesOwnedByService = executionBlueprint.getUnderlyingTypeNamesForService(service)
-
-        // it is MUCH quicker to compare membership in 2 sets rather than
-        // concat 1 giant set and then check
-        return objectTypeName in typeNamesOwnedByService
-            || objectTypeName in underlyingTypeNamesOwnedByService
-            || (executionContext.hints.sharedTypeRenames(service) && executionBlueprint.getUnderlyingTypeName(objectTypeName) in underlyingTypeNamesOwnedByService)
```

Comment:
I moved this above, and `sharedTypeRenames` isn't needed anymore with the new blueprint.